src/utils/ZDTypes.ts

```diff
@@ -0,0 +1,20 @@
+import {Dictionary} from 'mattermost-redux/types/utilities';
+
+export type ZDTriggerCondition = {
+    field: string;
+    operator: string;
+    value?: string;
+}
+
+export type ZDTriggerConditions = {
+    all?: ZDTriggerCondition[];
+    any: ZDTriggerCondition[];
```

Comment:
In the context of how the App currently operates, this type is only used with subscriptions and currently only supports the `any` field.  I added the `all` optional field because it will eventually be supported by the app.